drivers/neopixel/neopixel.py

```diff
@@ -1,44 +1,44 @@
 # NeoPixel driver for MicroPython
 # MIT license; Copyright (c) 2016 Damien P. George, 2021 Jim Mussared
 
-from micropython import const
 from machine import bitstream
 
-_BITSTREAM_TYPE_HIGH_LOW = const(0)
-_TIMING_WS2818_800 = (400, 850, 800, 450)
-_TIMING_WS2818_400 = (800, 1700, 1600, 900)
-
 
 class NeoPixel:
+    # G R B W
     ORDER = (1, 0, 2, 3)
 
-    def __init__(self, pin, n, bpp=3, timing=1):
+    def __init__(self, pin, n, bpp=3, t=1):
```

Comment:
I also think `timing` is part of the API (and you've actually documented it as a kw-only arg!).

Reply:
Updated branch. +5 bytes, now -71 net.